modules/common/go.mod

```diff
@@ -79,3 +79,7 @@ require (
 	github.com/kr/pretty v0.3.1 // indirect
 	github.com/rogpeppe/go-internal v1.10.0 // indirect
 )
+
+// mschuppert: map to latest commit from release-4.13 tag
+// must consistent with common module
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230414143018-3367bc7e6ac7
```

Comment:
Do we have significant difference between the old and the new version. Do we need to be careful bumping to this in the service operators? At least the keystone-operator bump seems to cause no issues so maybe this is not dangerous.

Reply:
there should be no issues, but we should update this. right now we use a really old OCP api version and there are no later releases, so we have to pin to a release tag. for the future we probably want to have versions/branches of our operators for each OCP release and then bump this to match the OCP release.

Reply:
and we have https://issues.redhat.com/browse/OSP-26293 to bump it anyway

Reply:
OK with me.